### DISCUSSION TITLE
When we use JMigrate on our browser? Did it happen to anyone that some of the embdded JS function gets ignored in these warnings?

### DISCUSSION THREAD

**Discussion by ashishpandey1091** [jquery/jquery-migrate#527] [Q&A] [answered]
For example,  if i have 
`$("someId").submit(); ` 
in a JS, and i run that in browser with JMigrate version, it does throw warnings.
But if i have something like 
`$(document).on("click", "someId", function() {
$("moreIds").submit();
});`
This warning gets ignored in JMigrate when i run the view in browser.

**Comment by dmethvin** [comment]
There's no code example, but if I understand your code snippet then the warning for the `.submit()` would appear on the console when you click `#someId`.

**Reply by ashishpandey1091** [reply]
Thank you so much for the quick response!

https://jsfiddle.net/ashish1091/tdxpey97/44/ 

I did create a fiddler to try this scenario, My concern was "moreIds" in the js code when i click on that button with JMigrate plugin, that didnt throw me a warning in browser console.

**Reply by dmethvin** [reply, marked as answer]
You're using an old version, 3.3.2. The latest version (3.5.2) gives the warnings I think you wanted. https://jsfiddle.net/hpy6ets5/

**Reply by ashishpandey1091** [reply]
Sure that makes sense. thank you so much! 
I was thinking if 3.3.2 Jmigrate version will also throw the warnings.
